### PR TITLE
Update of PPS/TOTEM data formats accessors

### PR DESCRIPTION
**Pull request by forthommel**
Dropped all get prefixes from PPS/TOTEM leaves, following https://github.com/cms-sw/cmssw/pull/28252.

### CODE REVIEW

**Comment by forthommel** [comment]
My favourite editor's hooks has unfortunately added random noise with its picky EOL empty spaces removal. Please let me know if you prefer to drop that cleaning part.

File: comparisons/validate.C [hunk 1]
```diff
@@ -1519,23 +1519,23 @@ void validateEvents(TString step, TString file, TString refFile, TString r="RECO
       tbr="TotemFEDInfos_totemRPRawToDigi_RP_"+recoS+".obj";
       if (checkBranchOR(tbr, true)){
         plotvar(tbr+"@.size()");
-        plotvar(tbr+".getFEDId()");
-        plotvar(tbr+".getOptoRxId()");
-        plotvar(tbr+".getFSize()");
+        plotvar(tbr+".fedId()");
```
Comment (slava77) [inline comparisons/validate.C]: new method names should be added, not replaced.
This is to keep this script functional on the old releases.

Comment (forthommel) [reply comparisons/validate.C]: Thanks @slava77 ! Added these methods back.

**Comment by slava77** [comment]
@smuzaffar 
this update looks fine for reco comparisons
please proceed with merging

**Comment by smuzaffar** [comment]
please test
let me re-run the tests as there are some untested changes

**Comment by slava77** [comment]
as a check for expected behavior, looking at the logs in all_OldVSNew_RunEGamma2018Awf136p85 I see that the reco comparisons script behaves as expected (the old version methods still used in the baseline work OK; while the newly added ones are reported as missing).

**Comment by slava77** [comment]
@smuzaffar 
please check this some time soon.
I have other more recent developments coming interfering with this PR.

**Comment by fabiocos** [comment]
+1

**Comment by fabiocos** [comment]
@smuzaffar @slava77 I will integrated cms-sw/cmssw#28252 in next IB

**Comment by smuzaffar** [comment]
+externals

**Pull request merged**